frontend/src/views/QuizCreateView.vue

```diff
@@ -0,0 +1,306 @@
+<template>
+  <div class="container">
+    <!-- 문제 내용 입력 컴포넌트 -->
+    <div class="content-field">
+      <label for="content" class="content">문제 내용</label>
+      <textarea type="text" :value="content" placeholder="문제를 입력하세요" @input="handleContentInput"
+        class="content-input" />
+    </div>
+    <hr />
+
+    <!-- 선택지 입력 컴포넌트들 -->
+    <div v-for="(option, index) in options" :key="index" class="option-field">
+      <div class="option-input-wrapper">
+        <!-- 선택지 입력란 -->
+        <input type="radio" :name="'option'" :value="index + 1" v-model="answer" :id="'option' + index" hidden />
+        <label :for="'option' + index" class="answer-button" :class="{ 'selected': answer === index + 1 }">
+          <i :class="answer === index + 1 ? 'bi bi-check-square' : 'bi bi-square'"></i>
+        </label>
+        <quiz-create-option :value="option" :placeholder="'선택지 ' + (index + 1) + '을 입력하세요'"
+          @input="handleOptionInput(index, $event)" />
+        <!-- 마지막 항목의 제거 버튼 -->
+        <i v-if="options.length > 2 && index === options.length - 1" @click="removeOption(index)"
+          class="bi bi-dash-square remove-button"></i>
+      </div>
+    </div>
+
+    <!-- 선택지 추가 버튼 -->
+    <i v-if="options.length < 4" class="bi bi-plus-lg add-button" @click="addOption"></i>
```

Comment:
`i` 태그는 `italic`의 약자로 주로 외국어나 용어 등을 강조할 때 사용해
여기서 특별히 이걸 사용한 이유가 없으면 `button`태그를 사용하는게 더 의미상 맞을거같아
```suggestion
    <button v-if="options.length < 4" class="bi bi-plus-lg add-button" @click="addOption"></button>
```
참고
[MDN Element <i>](https://developer.mozilla.org/ko/docs/Web/HTML/Element/i)

Reply:
bootstrap icon을 사용하여 icon의 의미로 `i` 태그를 사용했습니다.